core/types/bal/bal.go

```diff
@@ -83,77 +83,108 @@ func NewConstructionBlockAccessList() ConstructionBlockAccessList {
 	}
 }
 
+// EnableMutations enables the modification of the access list via subsequent
+// calls to record access events.  BALs are mutable by default.
+func (c *ConstructionBlockAccessList) EnableMutations() {
```

Comment:
These are leftover from when BALs did not include all state changes in the block (with tx EOA sender nonce increments being inferred).  This is no longer the case with the spec, and I will remove this logic.